zgw_consumers/models/services.py

```diff
@@ -64,6 +54,22 @@ class Service(models.Model):
         blank=True,
         help_text=_("Human readable representation of the user."),
     )
+    client_certificate = models.ForeignKey(
+        Certificate,
+        blank=True,
+        null=True,
+        help_text=_("The SSL/TLS certificate of the client"),
+        on_delete=models.CASCADE,
+        related_name="service_client",
+    )
+    server_certificate = models.ForeignKey(
+        Certificate,
+        blank=True,
+        null=True,
+        help_text=_("The SSL/TLS certificate of the server"),
+        on_delete=models.CASCADE,
```

Comment:
let's make this `models.PROTECT` instead - you don't want defined services to automatically be disabled if someone accidentally decides to "clean up" their certificates in the admin